src/fu-engine.c

```diff
@@ -1620,6 +1620,8 @@ fu_engine_get_report_metadata_kernel_cmdline (GHashTable *hash, GError **error)
 		g_auto(GStrv) tokens = fu_common_strnsplit (buf, bufsz - 1, " ", -1);
 		g_autoptr(GString) cmdline_safe = g_string_new (NULL);
 		for (guint i = 0; tokens[i] != NULL; i++) {
+			if (strlen (tokens[i]) == 0)
```

Comment:
I think that C99 requires that the variables be declared at start of the block.  So this probably needs to be:

```
g_auto(Gstrv) kv = NULL;
if (strlen ...)
kv = g_strsplit( ..)
```